script/setup

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/env sh
-
-# Set up Rails app. Run this script immediately after cloning the codebase.
-#
-# First, you need to create the database user the app will use by manually
-# typing the following in your terminal:
-#
-#   $ sudo -u postgres psql -c "CREATE USER ofn WITH SUPERUSER CREATEDB PASSWORD 'f00d'"
-#
-# This will create the "ofn" user as superuser and allowing it to create
-# databases.
-
-# Exit if any command fails
-set -e
-
-YELLOW='\033[0;33m'
-RED='\033[0;31m'
-NO_COLOR='\033[0m'
-
-RUBY_VERSION=$(cat .ruby-version)
-
-# Check ruby version
-if ! ruby --version | grep $RUBY_VERSION > /dev/null; then
-  printf "${RED}Open Food Network requires ruby ${RUBY_VERSION}${NO_COLOR}. Have a look at: https://github.com/rbenv/rbenv\n"
-  exit 1
-fi
-
-# Set up Ruby dependencies via Bundler
-if ! command -v bundle > /dev/null; then
-  ./script/install-bundler
-fi
-
-# Install all dependencies
-bundle check || bundle install
-yarn install
-
-# Set up the database for both development and test
-# Confirming the default user and password
-printf '\n\n' | bundle exec rake db:setup db:test:prepare
```

Comment:
The old script ran `db:test:prepare` to set up the test database too. From what [I can see](https://dev.to/software_writer/all-the-database-tasks-in-rails-2jic), `db:setup` won't do this. So I think it would be good to include this to help new starters.

Reply:
Right. We should only do that for dev and test environments though. I'm hoping that these scripts can be used during deployment one day to simplify the work of Ansible or Docker.